### ISSUE TITLE
installation details

### ISSUE THREAD

**Issue by dimpase** [gap-packages/numericalsgps#14]
IMHO it's not that easy to install "just anywhere", unless one is willing to modify GAP_ROOTS.
On the other hand at least on Unix one can install to `~/.gap/pkg/`

In fact I just wrote [this](https://github.com/sagemathinc/cocalc/issues/2333#issuecomment-423337354) explaining how to install the package on [cocalc](cocalc.com)

**Comment by pedritomelenas** [comment]
You can use `SetPackagePath("numericalsgps","yourpreferredpath");` and install gap in that path. You can add this to your gaprc file if you want. 

The file that mentioned this is already deleted, since NumericalSgps is now part of the gap distribution. 

Thank you for your comments,
Pedro

**Comment by dimpase** [comment]
You are not quite right to think that there is just one and only GAP distribution. Different distributions (Linux, Homebrew etc on OSX, etc) package different subsets of GAP packages.

**Comment by pedritomelenas** [comment]
I know this (I use gap in many ways and operating systems), and the procedure to install this package is like any other distributed package. For this, we decided not to integrate any instructions about it in README.

**Comment by dimpase** [comment]
Not all distributions package all the GAP packages. E.g. Sagemath does not package `numericalsgps` at the moment. I don't understand the reluctance not to include anything like [this](https://github.com/sagemathinc/cocalc/issues/2333#issuecomment-423337354) - unless you compete for a user-unfriendliness  award ;-)

**Comment by pedritomelenas** [comment]
@dimpase , probably you do not know me; people that know me would never say that I am competing for such an award. And I do not think that this should be part of the comments of an issue. The GAP manual already explains your procedure, and we posted in several places how to install gap with numericalsgps and helper packages. 
The point is that for the moment we decided to erase the file containing this info, and probably later we will write some instructions more suitable for the potential users of the package. 
For now we appended a paragraph pointing to the manual, where the user can also find the alternative to use `SetPackagePath`.
